doc/api.rst

```diff
@@ -26,21 +26,29 @@ ICLabel
 This is the model originally available for `EEGLab <https://github.com/sccn/ICLabel>`_.
 The model was ported from matconvnet using `pytorch <https://pytorch.org/>`_.
 
+ICLabel is designed to classify ICs fitted with an extended infomax ICA
+decomposition algorithm on EEG datasets referenced to a common average and
+filtered between [1., 100.] Hz.
+
 Architecture:
 
 .. image:: _static/ICLabel_DagNN_Architecture.png
    :width: 400
    :alt: ICLabel Neural Network Architecture
+   :align: center
 
-The model has three inputs: image, psd, and autocorrelation features. To encourage generalization, the image
-features are rotated and negated, thus quadrupling the feature. The psd and autocorrelation features
-are copied to the new image features. Then, the predicted probabilities are averaged over all four images.
+The model has three inputs: image (topomap), psd, and autocorrelation features.
+To encourage generalization, the image features are rotated and negated, thus
+quadrupling the feature. The psd and autocorrelation features are copied to the
+new image features. Then, the predicted probabilities are averaged over all four
+images.
```

Comment:
four or three images?

Reply:
Have a look at how I rephrase it, I don't really get what you meant by this sentence anyway 😅

Reply:
Oh haha I meant. Aren't the features comprised of 3 images? wdym by four image?

Reply:
The comment was clear, it's the sentence in the doc that is not 😅 
The PSD and autocorrelation are 1D: https://github.com/mne-tools/mne-icalabel/blob/0ae320a7c90cf7357547a34b4abf663511a1e0ad/mne_icalabel/iclabel/network.py#L52-L58 
Sure it's a Conv2D, but with a kernel size `(1, 3)`. So basically a Conv1D. Thus the terminology of `image` is already misleading.

Reply:
Or am I missing something?

Reply:
Ah I see. Yeah if I wrote this then I'm not sure what I be meant :p. 

Perhaps something like 3 features?

Reply:
That's my assumption as well :p 
Thus the rework as: 

> The model has three inputs: image (topomap), psd, and autocorrelation features.
> To encourage generalization, the image feature is rotated and negated, thus
> quadrupling the feature. After 3 convolutional layer with a ReLu activation,
> the 3 features are concatenated for the final layer.

Ok for you or would you like to change something else here?

Reply:
Yeah that looks good to me and more clear!